src/parallelization/hpxsimulator.cpp

```diff
@@ -6,12 +6,16 @@
 namespace LibGeoDecomp {
 namespace HpxSimulatorHelpers {
 
+hpx::lcos::local::promise<void> initPromise;
 std::map<std::string, hpx::lcos::local::promise<std::vector<double> > > localUpdateGroupSpeeds;
 std::map<std::string, hpx::lcos::local::promise<std::vector<double> > > myGlobalUpdateGroupSpeeds;
 std::map<std::string, hpx::lcos::local::promise<std::vector<std::size_t> > > myLocalityIndices;
 
 hpx::future<std::vector<double> > getUpdateGroupSpeeds(const std::string& basename)
```

Comment:
I know I should have asked this in the previous PR, but apparently did overlook this. What's the rationale here behind returning a future instead of the actual value?